src/routes/v1/capp_test.go

```diff
@@ -84,7 +84,7 @@ func TestPatchCapp(t *testing.T) {
 		Status: cappv1.CappStatus{},
 	}
 	body, _ := json.Marshal(cappRequest)
-	request, _ := http.NewRequest("PATCH", "/v1/namespaces/test-namespace/capps/test-capp", bytes.NewBuffer(body))
+	request, _ := http.NewRequest("PUT", "/v1/namespaces/test-namespace/capps/test-capp", bytes.NewBuffer(body))
 	request.Header.Set("Content-Type", "application/json")
```

Comment:
Create constants for these values

Reply:
Do you mean to the values in `Header.Set`?

Reply:
Yep